contrib/kml_export.lua

```diff
@@ -348,7 +365,7 @@ local function create_kml_file(storage, image_table, extra_data)
         else
             kmlFileOpenCommand = "xdg-open "..exportDirectory.."/\""..exportKMLFilename.."\""
 	end
-        dt.control.execute( kmlFileOpenCommand) 
+        coroutine.yield("RUN_COMMAND", kmlFileOpenCommand) 
```

Comment:
@elman22 I've added the script official/yield.lua. If you now download it and then require it in the first line of your luarc  the "dt.control.execute(command)" should work with your version of dt. 
If it works, please redo the changes from @boucman